docs/docs/docs/developer-resources/testing.md

```
@@ -503,3 +509,81 @@
+              "id": "ab1c2d3e-4f5b-6a7c-8d9e-0f1a2b3c4d5e",
+              "name": "Unity Foundation 1"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)